Markdown/cmm-2.0.14/system-operators/2-installation.md

```diff
@@ -652,14 +654,14 @@ For additional details, you can refer to the Monasca documentation. Help is also
 configuration settings. For a description of the settings, execute the following command:
 
 ```
-./monasca-agent-<version_number>.run --help
+./monasca-agent-3.0.3-CMM2.0.14-x.run --help
```

Comment:
Thanks for the update, \
Here I would rather leave the versions like:

- `monasca-agent-3.0.x-CMM2.0.14-x.run`

Because it is possible that we need to increment the version of the core libraries.